src/ide/completion/expr/macro_call.rs

```diff
@@ -22,7 +23,7 @@ pub fn macro_call_completions(
 
         if let Some(path) = expr_selector(db, &ctx.node);
         // Currently inline macros can not be imported/exported
-        if let [PathSegment::Simple(path_segment)] = path.segments(db).elements(db).as_slice();
+        if let [PathSegment::Simple(path_segment)] = path.segments(db).elements(db).collect_vec().as_slice();
```

Comment:
```suggestion
        if let [PathSegment::Simple(path_segment)] = path.segments(db).elements(db).take(2).collect_vec().as_slice();
```

No need to collect more now